interface/basisu_c_binding/src/basisu_c_binding.cpp

```diff
@@ -150,56 +137,53 @@ uint32_t basis_file::getImageTranscodedSizeInBytes(uint32_t image_index, uint32_
 
-    const transcoder_texture_format transcoder_format = static_cast<transcoder_texture_format>(format);
+    const transcoder_texture_format transcoder_format =
+        static_cast<transcoder_texture_format>(format);
 
     uint32_t orig_width, orig_height, total_blocks;
-    if (!m_transcoder.get_image_level_desc(m_file, byteLength, image_index, level_index, orig_width, orig_height, total_blocks))
+    if (!m_transcoder.get_image_level_desc(m_file, byteLength, image_index, level_index, orig_width,
+                                           orig_height, total_blocks))
         return 0;
 
-    uint32_t flags = get_alpha_for_opaque_formats ? cDecodeFlagsTranscodeAlphaDataToOpaqueFormats : 0;
+    uint32_t flags =
+        get_alpha_for_opaque_formats ? cDecodeFlagsTranscodeAlphaDataToOpaqueFormats : 0;
 
     uint32_t status;
 
-    if (basis_transcoder_format_is_uncompressed(transcoder_format))
-    {
-        MAYBE_UNUSED const uint32_t bytes_per_pixel = basis_get_uncompressed_bytes_per_pixel(transcoder_format);
+    if (basis_transcoder_format_is_uncompressed(transcoder_format)) {
+        MAYBE_UNUSED const uint32_t bytes_per_pixel =
+            basis_get_uncompressed_bytes_per_pixel(transcoder_format);
         MAYBE_UNUSED const uint32_t bytes_per_line = orig_width * bytes_per_pixel;
         MAYBE_UNUSED const uint32_t bytes_per_slice = bytes_per_line * orig_height;
 
         assert(bytes_per_slice <= dst_size);
 
         status = m_transcoder.transcode_image_level(
-            m_file, byteLength, image_index, level_index,
-            dst, orig_width * orig_height,
-            transcoder_format,
-            flags,
-            orig_width,
-            nullptr,
-            orig_height);
-    }
-    else
-    {
+            m_file, byteLength, image_index, level_index, dst, orig_width * orig_height,
+            transcoder_format, flags, orig_width, nullptr, orig_height);
+    } else {
         uint32_t bytes_per_block = basis_get_bytes_per_block_or_pixel(transcoder_format);
 
         MAYBE_UNUSED uint32_t required_size = total_blocks * bytes_per_block;
 
-        if (transcoder_format == transcoder_texture_format::cTFPVRTC1_4_RGB || transcoder_format == transcoder_texture_format::cTFPVRTC1_4_RGBA)
-        {
-            // For PVRTC1, Basis only writes (or requires) total_blocks * bytes_per_block. But GL requires extra padding for very small textures: 
+        if (transcoder_format == transcoder_texture_format::cTFPVRTC1_4_RGB ||
+            transcoder_format == transcoder_texture_format::cTFPVRTC1_4_RGBA) {
+            // For PVRTC1, Basis only writes (or requires) total_blocks * bytes_per_block. But GL
+            // requires extra padding for very small textures:
             // https://www.khronos.org/registry/OpenGL/extensions/IMG/IMG_texture_compression_pvrtc.txt
             // The transcoder will clear the extra bytes followed the used blocks to 0.
```

Comment:
Unrelated to this PR but given the overhead of running a PR please fix the typo. s/followed/following/.